tests/unit_tests/testdaemonaccesscontrol.h

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "helper.h"
+
+class TestDaemonAccessControl final : public TestHelper {
+  Q_OBJECT
+
+ private slots:
+  void initTestCase();
+  /**
+   * * "status" command is allowed for any peer when inactive
+   * * "activate" command is allowed for any peer when inactive
+   * * commands other than "activate" or "status" are not allowed
+   * is allowed for any peer when inactive.
+   *
+   */
+  void testInactiveCommands();
```

Comment:
Consider if names similar to testCommandsWhenInactive(), testAuthorizedPeerWhenActive() etc. are an improvement.